equinix/resource_fabric_network.go

```diff
@@ -0,0 +1,430 @@
+package equinix
+
+import (
+	"context"
+	"fmt"
+	v4 "github.com/equinix-labs/fabric-go/fabric/v4"
+	"github.com/equinix/terraform-provider-equinix/internal/config"
+	equinix_errors "github.com/equinix/terraform-provider-equinix/internal/errors"
+	equinix_fabric_schema "github.com/equinix/terraform-provider-equinix/internal/fabric/schema"
+	equinix_schema "github.com/equinix/terraform-provider-equinix/internal/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"log"
+	"strings"
+	"time"
+)
+
+func FabricNetworkChangeSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"href": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "href",
```

Comment:
Needs a better description. Refer to https://developer.equinix.com/docs?page=/dev-docs/fabric/overview